discovery-provider/plugins/notifications/src/processNotifications/mappers/userNotificationSettings.ts

```diff
@@ -136,7 +136,8 @@ export class UserNotificationSettings {
     return (
       this.userIsEmailDeliverable[receiverUserId] &&
       !isInitiatorAbusive &&
-      !userIsAbusive[receiverUserId]
+      !userIsAbusive[receiverUserId] &&
+      this.email?.[receiverUserId] !== 'off'
```

Comment:
we might want to check again for `userNotifications.email?.[track.owner_id].frequency === 'live'` since this is only used for live emails

Reply:
it should be used for scheduled emails too right? i just added it to processEmailNotifications